packages/shared/src/scanner/scanner.ts

```diff
@@ -94,6 +94,15 @@ export class Scanner {
 
             const scanStarted = new Date();
             const combinedScanResult = await this.crawler.crawl(crawlerParameters, this.baselineOptionsBuilder.build(scanArguments));
+
+            if (!isEmpty(combinedScanResult.errors)) {
+                this.logger.logInfo(`Scan failed with ${combinedScanResult.errors.length} error(s)`);
```

Comment:
What do you think about using `logError` here instead? With `logError` this message will elevate to the email notification and build summary.

```suggestion
                this.logger.logError(`Scan failed with ${combinedScanResult.errors.length} error(s)`);
```

Reply:
I like this better than the `logInfo` I was using. Good suggestion! I've updated to use `logError` instead. Thanks, Katy!